subprojects/dependency-management/src/main/java/org/gradle/internal/locking/LockFileReaderWriter.java

```diff
@@ -185,7 +185,7 @@ private void collectEmptyConfigurations(String line, List<String> emptyConfigura
     }
 
     private Path getUniqueLockfilePath() {
-        return lockFile.map(File::toPath).get();
+        return lockFile.map(file -> file.getAsFile().toPath()).get();
```

Comment:
:o: Out of curiosity, why, because this method is eager, not doing the simpler: `lockFile.get().getAsFile().toPath()`?

Reply:
Because it is a mistake 😉